modules/deploy/partials/kubernetes/install-crds.adoc

```diff
@@ -0,0 +1,14 @@
+ifdef::latest-operator-version[]
+[,bash,subs="attributes+"]
+----
+kubectl kustomize "https://github.com/redpanda-data/redpanda-operator//src/go/k8s/config/crd?ref={latest-operator-version}" \
+    | kubectl apply -f -
+----
+endif::[]
+ifndef::latest-operator-version[]
+[,bash]
+----
+kubectl kustomize "https://github.com/redpanda-data/redpanda-operator//src/go/k8s/config/crd" \
```

Comment:
Do you think it would be preferable to throw an error rather than falling back to the previous buggy template?

Alternatively, we could update this version to instead have a placeholder for the end user to fill out.

Reply:
Great idea. Will add the placeholder.